src/app.py

```diff
@@ -1,73 +1,354 @@
 import os
 import uuid
+from datetime import datetime, timedelta
 
+import jwt
 import pandas as pd
+from dateutil import parser as dt_parser
 from flask import Flask, abort, g, jsonify, request
+from flask_cors import CORS
 from loguru import logger
 
-from src.services.db.models2 import CatalogueItem, db
+from src.config import CONFIG_BY_ENV
+from src.services.db.enums import TransferStatus
+from src.services.db.models import CatalogueItem, db
+from src.services.db.schema import CatalogueItemSchema
+from src.utils import abort_json, clean_files, token_required
 
-DB_PORT = os.getenv("DB_PORT", 5432)
-DB_NAME = os.getenv("DB_NAME", "tempdb")
-DB_USER = os.getenv("DB_USER")
-DB_PASSWORD = os.getenv("DB_PASS")
-DB_URI = f"postgresql://{DB_USER}:{DB_PASSWORD}@localhost:{DB_PORT}/{DB_NAME}"
+ENV = os.getenv("FLASK_ENV", "local")
 
+# TODO: Raise error if values not set
+CFG = CONFIG_BY_ENV[os.getenv("FLASK_ENV", "local")]
+DB_URI = f"{CFG.DB_TYPE}://{CFG.DB_USER}:{CFG.DB_PASSWORD}@{CFG.DB_HOST}:{CFG.DB_PORT}/{CFG.DB_NAME}"
+
+logger.info("Starting the server...")
 app = Flask(__name__)
 app.config["DEBUG"] = True
```

Comment:
This should be environment dependent.

Reply:
Do you mean the name? or debug mode? (If debug mode, missed that part... Thanks for mentioning it...)